pyqgisserver/server.py

```diff
@@ -60,6 +61,15 @@ def read_configuration(argv: List[str]=None) -> argparse.Namespace:
     if args.version:
         print_version(verbose=args.debug)
         sys.exit(1)
+    elif args.version_all:
```

Comment:
Good idea, but I would move the whole stuff in `print_version()` and just keep only one `--version` option